lib/hostinfo.h

```diff
@@ -32,6 +32,16 @@
 #include "coproc.h"
 #include "common_defs.h"
 
+enum LINUX_OS_INFO_PARSER {
+    lsbrelease,
+    osrelease,
+    redhatrelease
+};
+
+const char command_lsbrelease[] = "/usr/bin/lsb_release -a 2>&1";
+const char command_osrelease[] = "/etc/os-release";
+const char command_redhatrelease[] = "/etc/redhat-release";
```

Comment:
Those are technically no commands but files which need to be opened. Only `lsb_release` is a command which needs to be executed. Since the difference is significant for the developer how to handle those the variable name should be correct: `file_osrealease` and `file_redhatrelease`.

Reply:
Fixed